profile/README.md

```diff
@@ -1 +1,45 @@
+
+## Code
+
+- [massa](https://github.com/massalabs/massa): Main code of Massa blockchain, builds the node and client, holds all the critical code for the architecture.
+- [massa-proto](https://github.com/massalabs/massa-proto): A repository centralizing protobuf bindings for Massa GRPC.
+- [massa-bridge](https://github.com/massalabs/massa-bridge): Blockchain bridge between Massa and any EVM-compatible blockchain.
+- [PeerNet](https://github.com/massalabs/PeerNet): A P2P network backend for Massa blockchain.
+- [stream_limiter](https://github.com/massalabs/stream_limiter): Rust library to implement read/write speed limits on any stream.
+
+## Smart contracts
+
+- [massa-sc-runtime](https://github.com/massalabs/massa-sc-runtime): Sandbox to run smart contracts (compiled as WASM), on a Massa node.
+- [massa-as-sdk](https://github.com/massalabs/massa-as-sdk): An SDK to build smart contracts in AssemblyScript. Abstracts Massa's ABI functions.
+- [massa-sc-toolkit](https://github.com/massalabs/massa-sc-toolkit): Provides tools such as `@massalabs/sc-project-initializer` to improve the smart-contract development experience.
+- [massa-sc-compiler](https://github.com/massalabs/massa-sc-compiler): Compile a smart contract to WASM binary.
+- [massa-sc-playground](https://github.com/massalabs/massa-sc-playground): Playground in AssemblyScript for smart-contracts.
+- [massa-web3](https://github.com/massalabs/massa-web3): A library to help the implementation of Web3 applications on top of Massa blockchain.
+- [massa-as-gaming-sdk](https://github.com/massalabs/massa-as-gaming-sdk): An SDK for smart contracts in AssemblyScript, made for game development.
+- [massa-standards](https://github.com/massalabs/massa-standards): Repository for smart-contracts standards like NFT and ERC-20, and all Massa standards.
+- [massa-sc-examples](https://github.com/massalabs/massa-sc-examples): Smart-contract examples.
+- [as](https://github.com/massalabs/as): Utils to write smart contracts in AssemblyScript.
+
+## Clients
+- [Thyra](https://github.com/massalabs/thyra): User-friendly client connected to the Massa blockchain to perform common operations. Accepts plugins to add more fonctionalities.
+- [thyra-plugin-store](https://github.com/massalabs/thyra-plugin-store): MassaLabs's list official Thyra plugins supported.
+- [thyra-plugin-wallet](https://github.com/massalabs/thyra-plugin-wallet): Plugin for Thyra implementing a wallet.
+- [thyra-plugin-blogNoCode](https://github.com/massalabs/thyra-plugin-blogNoCode): Plugin for Thyra to create a blog without coding, allowing to store the website on the blockchain.
+- [thyra-node-manager-plugin](https://github.com/massalabs/thyra-node-manager-plugin): Plugin for Thyra to manage a remote node from a user-friendly interface.
+- [thyra-playground-plugin](https://github.com/massalabs/thyra-playground-plugin): Plugin for Thyra to serve a playground for smart-contracts development.
```

Comment:
To delete or archive. Do you agree @longgamma-massa & @gregLibert?

Reply:
you want to delete the repo or the content of the readme ?